ERCS/erc-0.md

```diff
@@ -0,0 +1,178 @@
+---
+eip: 0
```

Comment:
```suggestion
eip: 8084
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.